.github/mergify.yml

```diff
@@ -0,0 +1,44 @@
+pull_request_rules:
+- name: auto-merge
+  description: automatic merge for main with > 1 approved reviews, all requested reviews have given feedback, not held, and CI is successful
+  conditions:
+    - "#approved-reviews-by>=1"
```

Comment:
I think we should set this to 2 to begin with and if we think that's too much friction bump it down to 1. I've been doing that with the PRs I've been reviewing over the last day and I think having that second set of eyes works well.

Reply:
That's not the org policy: https://github.com/instructlab/dev-docs/blob/main/docs/github-merge-strategy.md#approvals-from-project-maintainers

Reply:
Need to change the policy first, so I say proceed as-is and if we want to change the policy that needs to be a separate PR to dev-docs

Reply:
Imo The number of approved-by reviewers for a PR shouldn't be uniform across the entire org, it should be on a repo by repo basis especially since that repo isn't public. If I have to open up a separate PR there I will, but I think that's separate from the 1 vs 2 approved-by reviews decision.

Reply:
Yes - that was my thinking - enforce current policy, which then allows a PR workflow to propose and discuss changes from there

Reply:
and adding on -- I would expect maintainers to use their own judgment to decide when they feel additional reviews are needed. It's super easy to do. If you add another person to the list of requested reviewers, the PR can not merge until they all review it.